coalib/parsing/filters/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+This package holds filter objects. Filter objects are used to
+get the list of bears with specific properties.
+
+Example:
+    LanguageFilter: To get the list of bears for provided language/
```

Comment:
I wouldn't provide an example, I think people know how to handle the package roughly once they see its content ;)